### PR TITLE
Corrected cancel button functionality

### PR DESCRIPTION
**Pull request by Anvita0305**
**What kind of change does this PR introduce?**
Bugfix

**Issue Number:**
#1211 

**Did you add tests for your changes?**
N/A

**Snapshots/Videos:**
[Screencast from 20-12-23 09:04:25 AM IST.webm](https://github.com/PalisadoesFoundation/talawa-admin/assets/78889572/e62f5926-8e17-4eda-88fc-34bf4607ae51)

**If relevant, did you update the documentation?**
N/A

**Summary**
1. Corrected cancel button functionality
2. Fixed button's CSS

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: src/components/OrgListCard/OrgListCard.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ function orgListCard(props: InterfaceOrgListCardProps): JSX.Element {
 
   function handleClick(): void {
     const url = '/orgdash/id=' + _id;
-
+    console.log('clicked!');
```
Comment (noman2002) [inline src/components/OrgListCard/OrgListCard.tsx:29]: Remove logs.

**Review by noman2002 — CHANGES_REQUESTED** [review]

File: src/components/OrgListCard/OrgListCard.tsx (hunk 1 above)
Comment (Anvita0305) [reply src/components/OrgListCard/OrgListCard.tsx]: Done, please check.

**Review by noman2002 — CHANGES_REQUESTED** [review]
Please write the tests for missing lines.

**Review by noman2002 — CHANGES_REQUESTED** [review]
Please fix the failing tests.

**Comment by noman2002** [comment]
@Anvita0305 Any update on this ?

**Comment by Anvita0305** [comment]
Will start working this week

**Comment by palisadoes** [comment]
Please fix the conflicting files

**Comment by Anvita0305** [comment]
fixed conflicting files @palisadoes @noman2002

**Comment by palisadoes** [comment]
Please fix the conflicting file, it just came up

**Comment by palisadoes** [comment]
Please review:

- @rishav-jha-mech
- @aashimawadhwa
- @noman2002
- @ttaylor92

**Comment by Anvita0305** [comment]
Can someone please explain why this check is failing as I haven't done any changes to schema.graphql. @noman2002 
Screenshot 2024-01-15 023522

**Comment by palisadoes** [comment]
We discovered a bug in the introspection test. It will be fixed here
- https://github.com/PalisadoesFoundation/talawa-admin/issues/1429

**Comment by palisadoes** [comment]
Reviewers please review this PR. Ignore the failing introspection test for now.

**Comment by rishav-jha-mech** [comment]
Closing this due to inactivity

**Pull request closed without merging**